src/tools/data/datagen/CMakeLists.txt

```diff
@@ -343,29 +345,16 @@ ENDIF(SILO_FOUND)
 #  'ENV${PROGRAMFILES}'.
 #
 #-----------------------------------------------------------------------------
-IF(VISIT_USE_7ZIP)
-   SET(PF1 "PROGRAMFILES")
-   SET(PF2 "PROGRAMFILES(X86)")
-   FIND_PROGRAM(SEVEN_ZIP_EXE 7z "${VISIT_SEVEN_ZIP_DIR}" 
-                                 "$ENV{${PF1}}/7-Zip" 
-                                 "$ENV{${PF2}}/7-Zip")
-   UNSET(PF2)
-   UNSET(PF1)
-   IF("${SEVEN_ZIP_EXE}" STREQUAL "SEVEN_ZIP_EXE-NOTFOUND")
-       MESSAGE(WARNING "7-Zip not found, .tar.gz data files will not be extracted.")
-   ENDIF()
-ELSE(VISIT_USE_7ZIP)
-    FIND_PROGRAM(TAR_EXE tar)
-    IF(NOT "${TAR_EXE}" STREQUAL "TAR_EXE-NOTFOUND")
-        SET(TAR_CMD ${TAR_EXE})
-        SET(UNTAR_ARGS "-xzvf")
-        SET(TAR_ARGS "-czvf")
-    ELSE()
-        SET(TAR_CMD ${CMAKE_COMMAND})
-        SET(UNTAR_ARGS "-E tar -xzvf")
-        SET(TAR_ARGS "-E tar -czvf")
-    ENDIF()
-ENDIF(VISIT_USE_7ZIP)
+FIND_PROGRAM(TAR_EXE tar)
+IF(NOT "${TAR_EXE}" STREQUAL "TAR_EXE-NOTFOUND")
+    SET(TAR_CMD ${TAR_EXE})
+    SET(UNTAR_ARGS "xvf")
+    SET(TAR_ARGS "cvfJ")
+ELSE()
+    SET(TAR_CMD ${CMAKE_COMMAND})
+    SET(UNTAR_ARGS "-E tar xvf")
+    SET(TAR_ARGS "-E tar cvfJ")
+ENDIF()
```

Comment:
Windows system tar (at least my system) does not work with xz, so there should be logic to only use CMake on Windows.
Maybe `IF(NOT WIN32 AND NOT "${TAR_EXE}" ...`

Reply:
@biagas what do you think of the double negative on that line `NOT ${TAR_EXE} STREQUAL "TAR_EXE-NOT FOUND"` which amounts to *not not-found*? Gramamatically its bad but I wonder if there is a good CS reason to write the logic that way because changing to `${TAR_EXE} STREQUAL "TAR_EXE-NOT FOUND` will *match* only in the very specific case of that string.

Reply:
CMake's `if` actually understands the 'NOTFOUND', so you could rewrite as `if(TAR_EXE)`, which will be true if tar was found.

Reply:
Ok, I pushed c338762 for that